### ISSUE TITLE
Rails 6 get undefined method `assets` when new project

### ISSUE THREAD

**Issue by elct9620** [5xRuby/bankai#1]
Maybe breaks in our customize generator.

```ruby
/Users/elct9620/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/railties-6.0.2.1/lib/rails/railtie/configuration.rb:97:in `method_missing': undefined method `assets' for #<Rails::Application::Configuration:0x00007fa585297c28> (NoMethodError)
Did you mean?  asset_host
```

**Comment by elct9620** [comment]
Update: The `--api` options disable assets and break it.

**Comment by elct9620** [comment]
Fixed in v0.4.0
